movie-mlops-project/preprocessing/crawler.py

```
@@ -0,0 +1,113 @@
+        self._image_language = image_language
+        self._request_interval_seconds = request_interval_seconds
+    
+    def get_popular_movies(self, page):
+        params = {
+            "api_key": self._api_key,
+            "language": self._language,
+            "region": self._region,
+            "page": page
+        }
+        response = requests.get(f"{self._base_url}/popular", params=params)
+        if not response.status_code == 200:
```

Ruff (0.12.2)

Use `response.status_code != 200` instead of `not response.status_code == 200`

Replace with `!=` operator

(SIM201)